### ISSUE TITLE
Switch component does not apply an `isInvalid` style

### ISSUE THREAD

**Issue by wilson-webdev** [chakra-ui/chakra-ui#5064]
### Description

When the `isInvalid` prop is applied to the `Switch` component. I would expect some visual representation of this.

### Link to Reproduction

https://codesandbox.io/s/chakra-switch-isinvalid-efj7b?file=/src/index.tsx

### Steps to reproduce

1. Go to `index.tsx`
2. See that the component with `isInvalid` has no styling applied

### Chakra UI Version

1.6.12

### Browser

Google Chrome 95

### Operating System

- [X] macOS
- [ ] Windows
- [ ] Linux

### Additional Information

The [docs](https://chakra-ui.com/docs/form/switch#props) state that it updates the state of the unchecked state. Not sure if the component or the documentation needs to change.

**Comment by primos63** [comment]
First there is no styling for `_invalid` in the theme source. Therefore, I'm of the opinion the docs should be changed. I'm not even sure what `invalid` means for a switch. It's either on or off.  If it's an option that shouldn't be toggled because of other component settings then I'd expect the switch to be disabled. Would you agree?

**Comment by wilson-webdev** [comment]
Agreed. A `Checkbox` would be more applicable to applying an `isInvalid` state in this context. Docs should be changed to specify it doesn't update styling. 👍
